tests/test_tools.py

```diff
@@ -491,6 +492,41 @@ def test_is_dynamic_cwd_width():
         obs = is_dynamic_cwd_width(inp)
         yield assert_equal, exp, obs
 
+def test_is_tb_opt():
```

Comment:
Just give a more descriptive name. No need to save the character: `test_is_traceback_optional`